TypeCobol/Compiler/AntlrUtils/CobolErrorStrategy.cs

```diff
@@ -337,9 +293,11 @@ public class CodeElementErrorStrategy : CobolErrorStrategy
         /// </summary>
         protected override bool ErrorStrategyShouldNotConsumeThisToken(Token lastConsumedToken, Token nextToken)
         {
+            if (nextToken.TokenType == TokenType.SQL_FROM) return false;
```

Comment:
Do we need a special for `SQL_FROM` ? 
The test `SqlStatementElement.IsSqlStatementStartingKeyword(nextToken.TokenType)` should be enough, no ?

Reply:
Definitely ! Forgot that after writing it as a test.